cmd/launcher/main.go

```diff
@@ -45,6 +46,13 @@ func runMain() int {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// these systemslogger logs will go to stderr, which will dirty the results of our
+	// rundisclaimed operations. the only log we would see before running the subcommand
+	// should be "launcher starting up", so override here with a no-op logger
```

Comment:
Why do we lose the split between stdout and stderr?

Reply:
at least brew usage just uses the same output buffer for both, I could update but had assumed we did more of that or used `CombinedOutput` elsewhere enough to make this useful. happy to fix that up if you don't like this approach!

Reply:
I think we should aim to make this a generalized exec tool, and thus I think we should aim to preserve stderr stdout.

Reply:
makes sense to me! I will update